dags/transformation/dbt_nightly.py

```diff
@@ -73,30 +75,24 @@
     dag=seed_dag,
 )
 
-# dbt-run
-dbt_run_cmd = f"""
-    {dbt_install_deps_cmd} &&
-    SNOWFLAKE_TRANSFORM_WAREHOUSE=transform_l dbt run --profiles-dir profile --models tag:nightly
-"""
-
-dbt_seed_nightly = KubernetesPodOperator(
-    **pod_defaults,
-    image=DBT_IMAGE,
-    task_id="dbt-seed",
-    name="dbt-seed",
-    secrets=[
-        SNOWFLAKE_ACCOUNT,
-        SNOWFLAKE_USER,
-        SNOWFLAKE_PASSWORD,
-        SNOWFLAKE_TRANSFORM_ROLE,
-        SNOWFLAKE_TRANSFORM_WAREHOUSE,
-        SNOWFLAKE_TRANSFORM_SCHEMA,
-        SSH_KEY,
-    ],
-    env_vars=env_vars,
-    arguments=[dbt_install_deps_and_seed_cmd],
-    dag=dag,
-)
+# dbt_seed_nightly = KubernetesPodOperator(
+#     **pod_defaults,
+#     image=DBT_IMAGE,
+#     task_id="dbt-seed",
+#     name="dbt-seed",
+#     secrets=[
+#         SNOWFLAKE_ACCOUNT,
+#         SNOWFLAKE_USER,
+#         SNOWFLAKE_PASSWORD,
+#         SNOWFLAKE_TRANSFORM_ROLE,
+#         SNOWFLAKE_TRANSFORM_WAREHOUSE,
+#         SNOWFLAKE_TRANSFORM_SCHEMA,
+#         SSH_KEY,
+#     ],
+#     env_vars=env_vars,
+#     arguments=[dbt_install_deps_and_seed_cmd],
+#     dag=dag,
+# )
 
 # update_twitter_cmd = f"""
```

Comment:
@ifoukarakis, can this be removed?

Reply:
Doing a bit of "hygiene":

- Removing commented out code (we can use git history to find it if needed)
- Removing imports that are not needed.